mpd/base.py

```diff
@@ -319,6 +323,7 @@ def _parse_neighbors(self, lines):
         "crossfade",
         "delete",
         "deleteid",
+        "delpartition",
```

Comment:
Can you also add those commands here? https://github.com/Mic92/python-mpd2/blob/master/doc/topics/commands.rst

Reply:
Ok, I've updated that now. Let me know if I missed anything.